From 5cbcb9e389a02d6288f90a790c6b547d9f9dcac7 Mon Sep 17 00:00:00 2001
From: Drunkard Zhang <gongfan193@gmail.com>
Date: Tue, 28 Sep 2021 12:20:25 +0800
Subject: [PATCH] bugfix: add missed header

XDestroyImage is included in <X11/Xutil.h> which belongs to libX11.
Build fails without include this header on Gentoo.

Signed-off-by: Drunkard Zhang <gongfan193@gmail.com>
---
 src/framebuffer.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/framebuffer.cpp b/src/framebuffer.cpp
index 20909b3..da85a25 100644
--- a/src/framebuffer.cpp
+++ b/src/framebuffer.cpp
@@ -1,3 +1,4 @@
+#include <X11/Xutil.h>
 #include "framebuffer.hpp"
 
 slop::Framebuffer::Framebuffer( int w, int h ) {
